src/FittedImage.js

```diff
@@ -103,12 +107,20 @@ const FittedImage = React.createClass({
     this.setState({
       status: states.LOADED
     });
+
+    if (background || !modern) {
```

Comment:
Not sure you need the "if" ? I think the onLoad callback should always be called.